addon/utils/ce/raw-editor.ts

```diff
@@ -41,7 +41,7 @@ import RemoveTableCommand from "@lblod/ember-rdfa-editor/commands/remove-table-c
 @classic
 class RawEditor extends EmberObject {
   registeredCommands: Map<string, Command> = new Map()
-  protected model: Model;
+  model: Model;
```

Comment:
I believe we don't want model to be public, this probably means we should provide some wrappers on editor so the handler can get the selection.

Reply:
This was suggested by @abeforgit, we can just provide a getter for the selection as I think is a common use case